LondonBritishLibrary/orient/BLorient666.xml

```diff
@@ -0,0 +1,215 @@
+                            <idno>Wright 238</idno>
+
+                        </altIdentifier>
+
+                    </msIdentifier>
+                    <msContents>
+
+  <msItem xml:id="ms_i1">
+      <locus from="3ra" to="201va"/>
+      <title type="complete" ref="LIT4035SenkessarDL#FirstHalf"/>
+  <textLang mainLang="gez"/>
+      <note>Salām-hymns directly follow the commemorations to which they refer.</note>
+
+  <msItem xml:id="ms_i1.1">
+  <locus from="3ra" to="35rc"/>
+      <title type="complete" ref="LIT4035SenkessarDL#Maskaram"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.2">
+      <locus from="36ra" to="63vc"/>
+      <title type="complete" ref="LIT4035SenkessarDL#Teqemt"/>
+  </msItem>
+
+  <msItem xml:id="ms_i1.3">
+      <locus from="64ra" to="94rc"/>
+      <title type="complete" ref="LIT4035SenkessarDL#Hedar"/>
+      </msItem>
+
+  <msItem xml:id="ms_i1.4">
+      <locus from="95ra" to="134ra"/>
+      <title type="complete"  ref="LIT4035SenkessarDL#Tahsas"/>
+    </msItem>
+
+  <msItem xml:id="ms_i1.5">
+      <locus from="135ra" to="175va"/>
+      <title type="complete"  ref="LIT4035SenkessarDL#Terr"/>
+      </msItem>
+
+  <msItem xml:id="ms_i1.6">
+      <locus from="176ra" to="201va"/>
+      <title type="complete"  ref="LIT4035SenkessarDL#Yakkatit"/>
+    </msItem>
+  </msItem>
+
+                    </msContents>
+
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">202</measure>
+                                    <dimensions type="outer" unit="in">
+                                        <height>15.88</height>
+                                        <width>14</width>
+                                    </dimensions>
+                                </extent>
+                                
+                                <collation>
+                                    <note>The quire structure is very difficult to discern due to the tightness of the binding and the 
+                                    quality of the images available.</note>
+                                </collation>
+
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="35 38">
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <date notBefore="1667" notAfter="1682"/>
+                                <desc>Written in a fine character by two or three hands, or with several changes of nib.
```

Comment:
maybe, a writing tool? as a more universal term?

Reply:
Most probably it is copied by several hands. I see at least one break on fol. 151rb, around the midle of the column. Even though, the hands are not very different.